docs/sql-ref-syntax-qry-select-ignore-nulls.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: global
+title: IGNORE NULLS
+displayTitle: IGNORE NULLS 
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+
+### Description
+
+`IGNORE NULLS` is to ignore null values, which is used in first and last 
+
+### Syntax
+
+```sql
+[ first | last ] ( expression [ IGNORE NULLS ] ) ]
```

Comment:
Could we merge this syntax into `sql-ref-syntax-qry-select-groupby.md`?

Reply:
Merged~